modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/management_bar/components/filters/implementation/clientExtensionFilter.tsx

```diff
@@ -0,0 +1,188 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {ClientExtension} from 'frontend-js-components-web';
+import PropTypes from 'prop-types';
+import React, {useEffect, useState} from 'react';
+
+import type {FDSFilter} from '@liferay/js-api/data-set';
+
+import type {
+	FilterImplementation,
+	FilterImplementationArgs,
+	SetFilterArgs,
+} from '../Filter';
+
+export interface ClientExtensionFilterImplementationArgs
+	extends FilterImplementationArgs<unknown> {
+	cxFilterImplementation?: FDSFilter<unknown>;
+	cxFilterURL: string;
+}
+
+function getSelectedItemsLabel({
+	cxFilterImplementation,
+	cxFilterURL,
+	selectedData,
+}: ClientExtensionFilterImplementationArgs) {
+	if (!cxFilterImplementation) {
+		return '...';
```

Comment:
Why `...`? This is unusual fallback after sanity check, I would expect to see an empty string.

Reply:
Well, this should be defined by the design team, but I propose using `...` because that's the string shown to the user on top of the FDS to describe the filter. Since this `...` appears when the filter is not yet loaded, it makes more sense showing `...` than an empty string, because an empty string looks quite broken. But I agree `...` is only marginally better than an empty string and we should perhaps use a spinner.

Or, alternatively, we could deactivate the filter until it has been fully loaded.

Note however that this would only be used for preloaded data in a CX filter, something we haven't yet implemented. So, as the portal is now, this code won't ever be executed.

Reply:
Yeah, this sound like a problem for future us :+1:  So, this is fine, my only nit would be to extract this to a constant, something like `const LOADING_LABEL = '...'` .

Reply:
done